job/validators.py

```diff
@@ -277,6 +271,14 @@ def cross_clean_pv(pvmodel):
                                                   self.models["Site"].__getattribute__("longitude")):
                         self.add_validation_error("Wind", "Site",
                               "latitude/longitude not in the WindToolkit database. Cannot retrieve wind resource data.")
+            
+            if self.models["Settings"].off_grid_flag==True:
+                self.models["Wind"].operating_reserve_required_pct = 0.10
```

Comment:
@rathod-b Load`operating_reserve_required_pct` should not be overwritten if `off_grid_flag=True`. The default should be 0.1 but the user should be allowed to provide another value. Please adjust the validation to reflect this.

Reply:
Addressed in https://github.com/NREL/REopt_API/pull/343/commits/152ac98842eaaf16a7068f0412b34b0893f0886a